gradle/libs.versions.toml

```diff
@@ -1,6 +1,6 @@
 [versions]
-agp = "7.4.0"
-androidTools = "30.4.0" # == 23.0.0 + agp version
+agp = "8.0.0"
```

Comment:
We are updating layoutlib to the one that ships in studio flamingo and it's associated AGP 8.0.0 update. The previous update appeared to align those too.

This is just used in this project, it shouldn't impose anything on consumers of the paparazzi gradle plugin.

I do see now that I check that paparazzi has an `implementation` dependency on AGP instead of `compileOnly`, which should be fixed and I can look at doing that as part of this PR or a follow up.

Reply:
agp bumps sometimes need to accompany layoutlib bumps.  it really "depends" on whether the layoutlib bump has a dependency on sdk-common, etc. (ie.., layoutlib dependencies shared with agp) to avoid any need for classpath isolation/shadowing.

first try bumping layoutlib without the corresponding AGP bump and see if that makes anything easier.

re: implementation vs compileOnly, yea....it's a known issue.  see https://github.com/cashapp/paparazzi/issues/385

keeping a test matrix/range of compatibility with AGP is annoying and I've been kinda avoiding this, given the slew of issues that would eventually be filed when some API breaks.